docs/_docs/github-pages.md

```diff
@@ -5,102 +5,80 @@ permalink: /docs/github-pages/
 
 [GitHub Pages](https://pages.github.com) are public web pages for users,
 organizations, and repositories, that are freely hosted on GitHub's
-`github.io` domain or on a custom domain name of your choice. GitHub Pages are
-powered by Jekyll behind the scenes, so in addition to supporting regular HTML
-content, they’re also a great way to host your Jekyll-powered website for free.
+`github.io` domain or on a custom domain name of your choice. GitHub Pages are powered by Jekyll behind the scenes, so they're a great way to host your Jekyll-powered website for free.
```

Comment:
IMO, `GitHub Pages` is a singular entity.. so the sentence should **ideally** be `GitHub Pages is powered by..`

Also, it'd be better if the line was split at around 80 chars like elsewhere on this doc..